src/board.py

```diff
@@ -1,18 +1,58 @@
+from constants import N_ROWS, N_COLS
+
+
 def create_board():
-    return [[0 for j in range(5)] for i in range(5)]
+    return [[0 for j in range(N_COLS)] for i in range(N_ROWS)]
 
 
 def get_symbol(value):
     if value < 0:
-        return 'x'
+        return 'X'
     if value > 0:
-        return 'o'
+        return 'O'
     return ' '
 
 
-def print_board(board):
-    for line in board:
-        for tile in line:
-            symbol = get_symbol(tile)
-            print('[' + symbol + ']', end='')
-        print('')
+def print_board(board, instructions={}):
+    for x in range(len(board)):
+        for y in range(len(board[x])):
+            symbol = get_symbol(board[x][y])
+            instruction = instructions[(x, y)] if (x, y) in instructions else None
+            tileText = f'\t[{symbol}]' if instruction is None else f'\t[{symbol}]({instruction})'
+            print(f'{tileText}', end='\t')
```

Comment:
No need of literal string if you just send a variable.
Just write `print(tileText, end='\t')`